airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/up_to_date/commands.py

```diff
@@ -42,18 +42,23 @@
     default=False,
     help="Create a pull request.",
 )
-
+@click.option(
```

Comment:
Should we make this consistent with `migrate_to_poertry` and `migrate_to_base_image` so that they all have `--bump`?